src/qililab/digital/circuit_transpiler.py

```diff
@@ -110,7 +110,9 @@ def optimize_transpilation(self, nqubits: int, ngates: list[gates.Gate]) -> list
                 shift[gate.qubits[0]] += gate.parameters[0]
             # add CZ phase correction
             elif isinstance(gate, gates.CZ):
```

Comment:
We are also using gate for CZ instead of using from qibo.gates import ... (again I know that this is not related to your changes but I would like to point that out)